src/components/timeline/TimelineMarkerCanvas.js

```diff
@@ -90,6 +98,56 @@ class TimelineMarkerCanvas extends PureComponent {
     this.drawSeparatorsAndLabels(ctx, startRow, endRow);
   }
 
+  _drawOneMarker(
+    ctx: CanvasRenderingContext2D,
+    { x, y, w, h, text }: MarkerDrawingInformation,
+    colors?: { background: string, foreground: string } = {
+      background: '#9698C0',
+      foreground: 'black',
+    }
+  ) {
+    ctx.fillStyle = colors.background;
+
+    if (w >= h) {
+      // Ensure the text measurement tool is created, since this is the first time
+      // this class has access to a ctx.
+      if (!this._textMeasurement) {
+        this._textMeasurement = new TextMeasurement(ctx);
+      }
+      const textMeasurement = this._textMeasurement;
+
+      this.drawRoundedRect(ctx, x, y + 1, w, h - 1, 2);
```

Comment:
The last argument is now "2" because it's now a radius. "1" just doesn't look right.